tests/test_specs.py

```diff
@@ -304,25 +304,18 @@ async def test_openai_embedding_spec_with_missing_embeddings(openai_embedding_re
                 await ac.post("/v1/embeddings", json=openai_embedding_request_data, timeout=10)
 
 
+class TestOpenAIWithBatching(TestEmbedAPI):
+    def predict(self, batch):
+        assert len(batch) == 2, f"Batch size should be 4 but got {len(batch)}"
```

Comment:
Either check, or the message is incorrect.
You check if the length is 2, but the message says that length of 4 is expected

Reply:
nice catch, updating